corehq/apps/hqadmin/reports.py

```diff
@@ -1203,21 +1203,21 @@ def rows(self):
         now = datetime.utcnow()
         days = now - timedelta(days=90)
 
-        def get_data():
+        def get_data(domains):
             terms = [
                 AggregationTerm('domain', 'domain'),
                 AggregationTerm('commcare_version', 'form.meta.commcare_version')
             ]
-            query = FormES().submitted(gte=days, lte=now)
+            query = FormES().submitted(gte=days, lte=now).domain(domains)
             return NestedTermAggregationsHelper(base_query=query, terms=terms).get_data()
         rows = {}
         for domain in self.es_results.get('hits', {}).get('hits', []):
             domain_name = domain['_source']['name']
```

Comment:
You should also pass in `fields=['name']` to the `es_domain_query` above to prevent getting the whole domain doc. If you change that you'll also have to change this line to `domin_name = domain['fields']['name']`